### DISCUSSION TITLE
runDLKcat(): use DLKcat.tsv in a different directory

### DISCUSSION THREAD

**Discussion by PkiwiBird** [SysBioChalmers/GECKO#389] [Q&A] [answered]
It is possible to modify the directory where _DLKcat.tsv_ is stored, with:
`writeDLKcatInput(ecLiverMl,[],[],[],'path_2_light_DLKcat.tsv',true)`
but how to use that file as input for `runDLKcat()` ?
This is useful if we want to save the files of _DLKcat.tsv_ for the full and light models separately, as they are different.
`runDLKcat()` uses `modelAdapter.params`. 
Is there a way to run  `runDLKcat()`  using as input _DLKcat.tsv_  stored in a different directory than `modelAdapter.params`?

**Comment by edkerk** [comment, marked as answer]
Turned this into issue #392, as it should indeed be implemented. Will close  the discussion here, to continue on the issue page.
